.npmrc

```diff
@@ -0,0 +1 @@
+save-exact=true
```

Comment:
It's really minor and I don't feel confident with squashing / rebasing / whatever git magic it would require.